media/test-data.json

```diff
@@ -7,13 +7,15 @@
                 "name": "spring-maven",
                 "dir": "./spring-maven",
                 "type": "frontend",
+                "size": 5,
                 "preselected": false
             },
             {
                 "label": "Spring Gradle",
                 "name": "spring-gradle",
                 "dir": "./spring-gradle",
                 "type": "frontend",
+                "size": 9,
```

Comment:
is this change important?

Reply:
I updated the test file, so that we can test the operators. I would leave it in there

Reply:
Btw: You also can compare strings. Pretty cool

Reply:
ah sorry, I thought that you changed `5` to `9` but I just saw that these were to additions, no line removal